nautobot/docs/release-notes/version-2.0.md

```diff
@@ -63,7 +63,11 @@ Added the `?depth` query parameter in Nautobot v2.X to replace the `?brief` para
 
 Added Site Model Fields to Location. Location Model now has `asn`, `comments`, `contact_email`, `contact_name`, `contact_phone`, `facility`, `latitude`, `longitude`, `physical_address`, `shipping_address` and `time_zone` fields.
 
-### Changed
+### Added
+
+### Changed `ENABLE_NEW_UI_FEATURE` settings config ([#4668](https://github.com/nautobot/nautobot/issues/4668))
+
+Added a `ENABLE_ALPHA_UI` configuration option to the settings, which is initially set to False. When set to True, this option enables the NewUI Feature.
```

Comment:
Rather than a separate section, let's add this under the existing "Alpha UI 2.0" section in a `+/- 2.0.3` block, something like:

```markdown
#### Alpha UI 2.0 ([#3142](https://github.com/nautobot/nautobot/issues/3142))

Nautobot 2.0 includes an "alpha" version of a new user interface (UI) for Nautobot, based on the React web framework.

+/- 2.0.3
    This UI is disabled by default but can be enabled as an option by setting the `ENABLE_ALPHA_UI` setting to `True` in your `nautobot_config.py` and then running [`nautobot-server build_ui`](../user-guide/administration/tools/nautobot-server.md).
```